src/styles/components/_form.scss

```
@@ -36,34 +38,63 @@
   &__input-label {
     display: block;
     margin-bottom: rem(12px);
+
+    font-size: rem(16px);
+    font-weight: 600;
+    line-height: 1.2;
   }
 
-  &__input-group {
+  &__group {
     width: 100%;
   }
 
-  &__actionable,
-  &__recaptcha {
+  &__input-password {
+    position: relative;
     display: flex;
-    justify-content: center;
+    flex-direction: column;
+    box-sizing: border-box;
   }
 
   &__input {
-    width: 100%;
+    display: block;
     box-sizing: border-box;
-    padding: rem(4px 12px);
+    width: 100%;
+    border-width: 0 0 rem(2px);
+    border-color: $havelock-blue;
```

Alphabetically, `c` comes before `w`
```suggestion
    border-color: $havelock-blue;
    border-width: 0 0 rem(2px);
```